src/content/docs/en/guides/astro-db.mdx

```diff
@@ -654,6 +655,68 @@ When you're ready to deploy, see our [Deploy with a Studio Connection guide](#de
 
 </ReadMore>
 
+## LibSQL
+
+<p><Since pkg="@astrojs/db" v="0.14.0" /></p>
+
+Astro DB can connect to any LibSQL server from any platform that exposes the Hrana protocol of the server, or can be self-hosted.
```

Comment:
```suggestion
Astro DB can connect to any libSQL server from any platform that exposes the libSQL remote protocol of the server, or can be self-hosted.
```

We're moving away from calling it "Hrana", so not using it here would be a huge help 🫶

Reply:
As a reader, this also makes it a lot easier to understand what it actually is in this context as well.

Reply:
The name everywhere I see is LibSQL with capital L, is that changing too?

Reply:
I looked again, and there seems to be a mix. I'll normalize our references to be `libSQL` but as a heads up @notrab , on Turso docs and the repos the links and titles are mostly using "LibSQL" while the content on the pages are using "libSQL".